src/dynatrace/opentelemetry/metrics/export/_exporter.py

```diff
@@ -0,0 +1,146 @@
+# Copyright 2020 Dynatrace LLC
```

Comment:
```suggestion
# Copyright 2022 Dynatrace LLC
```
I think new files get the year in which they were first released as their Copyright year

Reply:
fixed in [0ce3ad9](https://github.com/dynatrace-oss/opentelemetry-metric-python/pull/27/commits/0ce3ad925520917615a5767d9205f81dda0762a5) :slightly_smiling_face: